cms/api.py

```diff
@@ -35,3 +37,15 @@ def get_course_enrollment_text(course):
             text = 'Coming ' + course_run.fuzzy_start_date
 
     return text
+
+
+def get_course_url(course):
+    """
+    Construct the course page url
+    """
+    course_run = course.get_next_run()
+    if course_run:
+        course_key = course_run.edx_course_key
+        url = settings.EDXORG_BASE_URL
+        return "{}/courses/{}/about".format(url, course_key)
```

Comment:
Can you use `urljoin` here to join the edX base URL and the rest of the path? String concat is fine for the `courses/{}/about` part of the URL